appcelerator/titanium_mobile
# refactor(android)(9_3_X): removed unneeded libraries

**JIRA:**
- https://jira.appcelerator.org/browse/TIMOB-27873
- https://jira.appcelerator.org/browse/TIMOB-28081

**Summary:**
- Removed Apache "common-logging" library.
  * We were not using this library at all. (Perhaps many years ago we used to.)
- Removed "nineoldandroids" library.
  * Was used to implement animations on Android 2.3 and older.
  * Switched to using Android OS built-in animation classes.

**Test:**
1. Create a "Classic" Titanium app project.
2. Add the below "build.gradle" file to project folder: `./platform/android`
3. Copy the below "app.js" code.
4. Build and run on Android.
5. Verify it builds successfully. (This tests [TIMOB-28081](https://jira.appcelerator.org/browse/TIMOB-28081).)
6. Verify you see a blue square with a black border scale in/out on-screen.

./platform/android/build.gradle
```groovy
dependencies {
	implementation 'commons-logging:commons-logging:1.1.3'
}
```

./Resources/app.js
```javascript
var window = Ti.UI.createWindow({
	backgroundColor: "white",
	fullscreen: true,
});
var squareView = Ti.UI.createView({
	backgroundColor: "blue",
	borderColor: "black",
	borderWidth: "2dp",
	width: "80dp",
	height: "80dp",
});
window.add(squareView);
window.addEventListener("postlayout", function onLayout() {
	window.removeEventListener("postlayout", onLayout);
	var windowSize = window.size;
	var minDimension = Math.min(windowSize.width, windowSize.height);
	var letterboxScale = minDimension / squareView.size.width;
	var animation = Ti.UI.createAnimation({
		transform: Ti.UI.createMatrix2D().scale(letterboxScale, letterboxScale),
		duration: 2000,
		autoreverse: true,
		repeat: 1000,
	});
	squareView.animate(animation);
});
window.open();
```
